modules/distr-tracing-tempo-monolithic-config-receiver-tls.adoc

```diff
@@ -0,0 +1,73 @@
+:_mod-docs-content-type: REFERENCE
+[id="distr-tracing-tempo-config-receiver-tls_{context}"]
+= Configure receiver TLS
+
+The `TempoMonolithic` custom resource (CR) allow to configure TLS on the distributor receivers when is not used with the gateway. You can specify a certificate, first you need to create a Secret with the tls.key and tls.crt keys, Those keys will contains the certificate and the key. Optionally you can specify the CA in a ConfigMap if you want to enable mutual TLS authentication.
```

Comment:
By "You can specify a certificate" do you mean
this
https://docs.openshift.com/container-platform/4.14/security/certificate_types_descriptions/user-provided-certificates-for-default-ingress.html#cert-types-user-provided-certificates-for-default-ingress
or
this
https://docs.openshift.com/container-platform/4.14/security/certificate_types_descriptions/user-provided-certificates-for-api-server.html#cert-types-user-provided-certificates-for-the-api-server
?

Reply:
None of those I think, we have specified fields on the CRD to set the configmap and secrets for the custom certificates.